misaligned.c

```diff
@@ -10,12 +12,15 @@ int printColorMap() {
             printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);
         }
     }
+    
+    TotalColorpairPrinted = ((i-1)*5)+(j-1);
     return i * j;
 }
 
 int main() {
     int result = printColorMap();
     assert(result == 25);
+    assert(result == TotalColorpairPrinted);

```

Comment:
Make the assert to reflect the functionality. This assert says 'print 25 items'. Sharpen it to reflect another aspect of the reference manual